.travis.yml

```diff
@@ -18,17 +18,17 @@ env:
   - PUPPET_VERSION="~> 4.3.0"
 matrix:
   exclude:
-  - rvm: 2.2.3
```

Comment:
All of these conflict with https://github.com/dfarrell07/puppet-opendaylight/pull/120